### DISCUSSION TITLE
Error creating app using "Vacation request form Template"

### DISCUSSION THREAD

**Discussion by berief** [Budibase/budibase#4602] [Help]
Hello,

we want to implement the Vacation request Template in our company. But we tried to use it a few times in different ways, but its not working. Always the following Error: "Error creating app"

Thank you for the great templates

**Comment by mjashanks** [comment]
Hey @berief  - I am experiencing the same issue. Thanks for reporting - we'll get this looked at!

Logged here: https://github.com/Budibase/budibase/issues/4606
